src/test/objectstore/store_test.cc

```diff
@@ -2512,7 +2512,7 @@ TEST_P(StoreTest, SimpleMoveTempTest) {
   ObjectStore::Sequencer osr("test");
   int r;
   coll_t cid, cid2;
- if ((GetParam() == string("filestore")) ) {
+  if ((GetParam() == string("filestore")) ) {
```

Comment:
true. currently only kstore and filestore implementations are present. so i have added that condition. added kstore param here. after bluestore and other implementations, we can remove this condition altogether.